docs/change-log/index.md

```diff
@@ -74,8 +74,9 @@ To install on a **Python 3.7** environment (and optionally upgrade to python 3.9
 2. Within the Jupyter service, open a terminal and update conda and pip to have an up-to-date pip resolver.
 
 ```
-$CONDA_HOME/bin/conda install -y conda=23.1.0
+$CONDA_HOME/bin/conda install -y conda=23.1.0 
 $CONDA_HOME/bin/conda install -y 'pip>=22.0'
+$CONDA_HOME/bin/conda install -y nbformat=5.7.0
```

Comment:
please add a note above that there is a known bug with nbformat on iguazio <= 3.5.2 jupyter, therfore it must be upgraded to 5.7.0
the bug is that some jupyter fails to open some notebooks with old nbformat